src/utils/GraphhopperUtils.js

```diff
@@ -121,22 +166,41 @@ async function fetchRoutes(end: string, start: string, departureTimeDateNow: str
   let walkingRoute;
 
   const sharedOptions = { method: 'GET', qsStringifyOptions: { arrayFormat: 'repeat' } };
-  const busOptions = {
-    qs: getGraphhopperBusParams(end, start, departureTimeDateNow, isArriveByQuery),
+  // Fetch bus routes using the current start time
+  const busOptionsNow = {
+    qs: getGraphhopperBusParams(end, start, departureTimeDateNow, isArriveByQuery, 0),
     url: `http://${GHOPPER_BUS || 'ERROR'}:8988/route`,
     ...sharedOptions,
   };
+
+  // Fetch bus routes using a delay buffer of DELAY_BUFFER_IN_MINUTES.
+  // This means that we are fetching routes with the start time as DELAY_BUFFER_IN_MINUTES
+  // ago from the current start time.
```

Comment:
Maybe  just`Fetch routes with (startTime - DELAY_BUFFER_IN_MINUTES)`, and that's more clear?